modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/impl/MotechSchedulerDatabaseServiceImpl.java

```diff
@@ -103,14 +105,24 @@ public int countJobs(JobsSearchSettings jobsSearchSettings) throws MotechSchedul
         }
     }
 
-
+    private String getQueryForEmptyFilter(JobsSearchSettings jobsSearchSettings){
```

Comment:
it is more readable to use one variable, assign value depends on if condition and then return this value